src/components/HistoryItemDetails/HistoryItemDetailsInternal.js

```diff
@@ -37,8 +37,14 @@ export class HistoryItemDetailsInternal extends PureComponent<Props> {
       return null
     }
 
+    const extendedProps = {
+      ...this.props,
+      editNote: this.debouncedHandlerOnEditNote,
+      key: this.props.id,
+    }
```

Comment:
Does flow able to check such structures?
Probably better to define specific type for it.

Reply:
haha! Already done, but not pushed to repo. I'm playing with it right now :)